code/cssgrid/style.css

```diff
@@ -0,0 +1,49 @@
+#wrapper {
+    font-size: 2em;
+    gap: 1rem;
+    display: grid;
+    grid-template-areas: 
+    "header header header"
+    "left main right"
+    "footer footer footer";
+    grid-template-columns: 25% 60% auto;
```

Comment:
The instructions were for the left column to be a fixed size, whereas %'s change with the viewport. 

I'm still marking this as complete though, good work!